### DISCUSSION TITLE
Implement JSI for status-go instead of RN bridge

### DISCUSSION THREAD

**Discussion by flexsurfer** [status-im/status-legacy#16628] [General]
# Feature Issue

Currently, status-go is implemented as a native module and uses RN bridge, when we load messages its a lot of data, so the bridge is a bottleneck, there is a better way, React Native JSI Modules

### Description

https://blog.notesnook.com/getting-started-react-native-jsi/

**Comment by churik** [comment]
@flexsurfer still the case?

**Comment by flexsurfer** [comment]
yes

**Comment by cammellos** [comment]
This will require a bit more discussion and it's not actionable at the moment, moving to discussion for now
